asd-aituber/apps/web/__tests__/hooks/usePerformanceMonitor.test.ts

```
@@ -0,0 +1,430 @@
+  memory: {
+    usedJSHeapSize: 50 * 1024 * 1024,
+    totalJSHeapSize: 100 * 1024 * 1024,
+    jsHeapSizeLimit: 200 * 1024 * 1024,
+  },
+  now: vi.fn(() => Date.now()),
+  mark: vi.fn(),
+  measure: vi.fn(),
+}
+
+class MockPerformanceObserver {
+  constructor(callback: PerformanceObserverCallback) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)